content/slice/language-guide/enum-types.md

```diff
@@ -206,13 +335,66 @@ public static class FruitSliceDecoderExtensions
     public static Fruit DecodeFruit(this ref SliceDecoder decoder) => ...
 }
 ```
+
+For enums with an underlying type, the Slice compiler also generates an extension method As*Name* to convert a value
+of the underlying type into an enumerator.
+
+With our Fruit example:
+
+```csharp
+public static class FruitByteExtensions
+{
+    public static Fruit AsFruit(this byte value) => ...
```

Comment:
Should this be int as well?

```suggestion
    public static Fruit AsFruit(this int value) => ...
```

Reply:
Looks like the `Fruit` enum uses `uint8` as it's underlying type to me.

Reply:
And uint8 maps to byte.